client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table/index.tsx

```diff
@@ -39,77 +47,125 @@ const SiteTable = ( { isLoading, columns, items }: Props, ref: Ref< HTMLTableEle
 		showLicenseInfo( license );
 	};
 
-	return (
-		<table ref={ ref } className="site-table__table">
-			<thead>
-				<tr>
-					{ isBulkManagementActive ? (
-						// Take full-width of the table
-						<th colSpan={ 100 }>
-							<SiteBulkSelect isLargeScreen sites={ items } isLoading={ isLoading } />
-						</th>
-					) : (
-						<>
-							{ columns.map( ( column, index ) => (
-								<th key={ column.key }>
-									<SiteSort isLargeScreen isSortable={ column.isSortable } columnKey={ column.key }>
-										{ index === 0 && (
-											<Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />
-										) }
-										<span className={ classNames( index === 0 && 'site-table-site-title' ) }>
-											{ column.title }
+	useEffect( () => {
+		const urlParams = new URLSearchParams( window.location.search );
+		if ( urlParams.get( 'tour' ) === 'add-new-site' ) {
+			dispatch(
+				savePreference( 'jetpack-cloud-site-dashboard-add-new-site-tour-site-count', items.length )
```

Comment:
Perhaps we should save the count in `localStorage` instead of a permanent preference?

Reply:
Done in 401c4d0.